### PR TITLE
[NOT AZURE] Azure Stack Backup Admin SDK

### PR DESCRIPTION
**Pull request by deathly809**
## Description

This is for Azure Stack **ONLY**.  There seems to be an issue when generating the SDK remotely for the repo some reason.  I have message the Autorest team and ask them for help but no response.

https://github.com/Azure/azure-rest-api-specs/pull/2838

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [X] Please add REST spec PR link to the SDK PR
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [X] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [X] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [X] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by dsgouda** [comment]
@deathly809 please fix test failures here

**Comment by deathly809** [comment]
@dsgouda What does "Queue empty" mean?  It looks like it is coming from the test framework but I don't know what to do about it.

**Comment by dsgouda** [comment]
Taking a look.

**Comment by deathly809** [comment]
@dsgouda It took me over an hour to record that test.  I am still not sure why the old recording was not working.

**Comment by deathly809** [comment]
@dsgouda For some reason the Batch tests broke.  I did not touch those at all.

File: src/SDKs/_metadata/azsadmin_resource-manager_backup.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+﻿Executing AutoRest command
+cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/azsadmin/resource-manager/backup/readme.md --csharp --csharp-sdks-folder=E:\github\azure-sdk-for-net\src\AzureStack\BackupAdmin\Backup.Admin --version=latest --reflect-api-versions
+
+AutoRest code generation for https://github.com/Azure/azure-rest-api-specs/blob/master/specification/azsadmin/resource-manager/backup/readme.md failed. Please try again
```
Comment (dsgouda) [inline src/SDKs/_metadata/azsadmin_resource-manager_backup.txt]: looks like code generation failed here, please check in code generated by generate.ps1

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please check in code generated by either generate.cmd or generate.ps1

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**